### ISSUE TITLE
Unable to automatically update an API with a mediation

### ISSUE THREAD

**Issue by RAVEENSR** [wso2/product-apim-tooling#481]
**Description:**

When trying to update an API using the import command, gives the following error:

```
[2020-09-18 18:41:50,766] ERROR - APIUtil Issue is in accessing the Registry

[2020-09-18 18:41:50,766] ERROR - APIGatewayManager Error in updating the sequence at the Gateway

org.wso2.carbon.apimgt.api.APIManagementException: Issue is in accessing the Registry

	at org.wso2.carbon.apimgt.impl.utils.APIUtil.getCustomSequence_aroundBody264(APIUtil.java:4606)

	at org.wso2.carbon.apimgt.impl.utils.APIUtil.getCustomSequence(APIUtil.java:4547)

	at org.wso2.carbon.apimgt.impl.APIGatewayManager.deployInSequence_aroundBody28(APIGatewayManager.java:719)

	at org.wso2.carbon.apimgt.impl.APIGatewayManager.deployInSequence(APIGatewayManager.java:713)

	at org.wso2.carbon.apimgt.impl.APIGatewayManager.updateCustomSequences_aroundBody34(APIGatewayManager.java:829)

	at org.wso2.carbon.apimgt.impl.APIGatewayManager.updateCustomSequences(APIGatewayManager.java:798)

	at org.wso2.carbon.apimgt.impl.APIGatewayManager.publishToGateway_aroundBody2(APIGatewayManager.java:156)

	at org.wso2.carbon.apimgt.impl.APIGatewayManager.publishToGateway(APIGatewayManager.java:90)

	at org.wso2.carbon.apimgt.impl.APIProviderImpl.publishToGateway_aroundBody82(APIProviderImpl.java:1749)

	at org.wso2.carbon.apimgt.impl.APIProviderImpl.publishToGateway(APIProviderImpl.java:1733)

	at org.wso2.carbon.apimgt.impl.APIProviderImpl.updateAPI_aroundBody58(APIProviderImpl.java:1110)

	at org.wso2.carbon.apimgt.impl.APIProviderImpl.updateAPI(APIProviderImpl.java:978)

	at org.wso2.carbon.apimgt.impl.UserAwareAPIProvider.updateAPI_aroundBody6(UserAwareAPIProvider.java:88)

	at org.wso2.carbon.apimgt.impl.UserAwareAPIProvider.updateAPI(UserAwareAPIProvider.java:83)

	at org.wso2.carbon.apimgt.importexport.utils.APIImportUtil.updateAPI(APIImportUtil.java:672)

	at org.wso2.carbon.apimgt.importexport.APIService.updateAPI(APIService.java:352)

	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)

	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)

	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)

	at java.lang.reflect.Method.invoke(Method.java:498)

	at org.apache.cxf.service.invoker.AbstractInvoker.performInvocation(AbstractInvoker.java:179)

	at org.apache.cxf.service.invoker.AbstractInvoker.invoke(AbstractInvoker.java:96)

	at org.apache.cxf.jaxrs.JAXRSInvoker.invoke(JAXRSInvoker.java:193)

	at org.apache.cxf.jaxrs.JAXRSInvoker.invoke(JAXRSInvoker.java:103)

	at org.apache.cxf.interceptor.ServiceInvokerInterceptor$1.run(ServiceInvokerInterceptor.java:59)

	at org.apache.cxf.interceptor.ServiceInvokerInterceptor.handleMessage(ServiceInvokerInterceptor.java:96)

	at org.apache.cxf.phase.PhaseInterceptorChain.doIntercept(PhaseInterceptorChain.java:308)

	at org.apache.cxf.transport.ChainInitiationObserver.onMessage(ChainInitiationObserver.java:121)

	at org.apache.cxf.transport.http.AbstractHTTPDestination.invoke(AbstractHTTPDestination.java:267)

	at org.apache.cxf.transport.servlet.ServletController.invokeDestination(ServletController.java:234)

	at org.apache.cxf.transport.servlet.ServletController.invoke(ServletController.java:208)

	at org.apache.cxf.transport.servlet.ServletController.invoke(ServletController.java:160)

	at org.apache.cxf.transport.servlet.CXFNonSpringServlet.invoke(CXFNonSpringServlet.java:216)

	at org.apache.cxf.transport.servlet.AbstractHTTPServlet.handleRequest(AbstractHTTPServlet.java:301)

	at org.apache.cxf.transport.servlet.AbstractHTTPServlet.doPut(AbstractHTTPServlet.java:237)

	at javax.servlet.http.HttpServlet.service(HttpServlet.java:653)

	at org.apache.cxf.transport.servlet.AbstractHTTPServlet.service(AbstractHTTPServlet.java:276)

	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)

	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)

	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)

	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)

	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)

	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:219)

	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:110)

	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:603)

	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:169)

	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:104)

	at org.wso2.carbon.identity.context.rewrite.valve.TenantContextRewriteValve.invoke(TenantContextRewriteValve.java:80)

	at org.wso2.carbon.identity.authz.valve.AuthorizationValve.invoke(AuthorizationValve.java:100)

	at org.wso2.carbon.identity.auth.valve.AuthenticationValve.invoke(AuthenticationValve.java:65)

	at org.wso2.carbon.tomcat.ext.valves.CompositeValve.continueInvocation(CompositeValve.java:99)

	at org.wso2.carbon.tomcat.ext.valves.CarbonTomcatValve$1.invoke(CarbonTomcatValve.java:47)

	at org.wso2.carbon.webapp.mgt.TenantLazyLoaderValve.invoke(TenantLazyLoaderValve.java:57)

	at org.wso2.carbon.event.receiver.core.internal.tenantmgt.TenantLazyLoaderValve.invoke(TenantLazyLoaderValve.java:48)

	at org.wso2.carbon.tomcat.ext.valves.TomcatValveContainer.invokeValves(TomcatValveContainer.java:47)

	at org.wso2.carbon.tomcat.ext.valves.CompositeValve.invoke(CompositeValve.java:62)

	at org.wso2.carbon.tomcat.ext.valves.CarbonStuckThreadDetectionValve.invoke(CarbonStuckThreadDetectionValve.java:159)

	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:1025)

	at org.wso2.carbon.tomcat.ext.valves.CarbonContextCreatorValve.invoke(CarbonContextCreatorValve.java:57)

	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)

	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:445)

	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1137)

	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:637)

	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1775)

	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.run(NioEndpoint.java:1734)

	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)

	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)

	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)

	at java.lang.Thread.run(Thread.java:748)

Caused by: org.wso2.carbon.registry.core.exceptions.ResourceNotFoundException: Resource does not exist at path /_system/governance/apimgt/applicationdata/provider/admin/MyT/1.0.0/in/

	at org.wso2.carbon.registry.core.jdbc.EmbeddedRegistry.get(EmbeddedRegistry.java:532)

	at org.wso2.carbon.registry.core.caching.CacheBackedRegistry.getGhostResourceFromCache(CacheBackedRegistry.java:225)

	at org.wso2.carbon.registry.core.caching.CacheBackedRegistry.get(CacheBackedRegistry.java:198)

	at org.wso2.carbon.registry.core.session.UserRegistry.getInternal(UserRegistry.java:617)

	at org.wso2.carbon.registry.core.session.UserRegistry.access$400(UserRegistry.java:61)

	at org.wso2.carbon.registry.core.session.UserRegistry$5.run(UserRegistry.java:597)

	at org.wso2.carbon.registry.core.session.UserRegistry$5.run(UserRegistry.java:594)

	at java.security.AccessController.doPrivileged(Native Method)

	at org.wso2.carbon.registry.core.session.UserRegistry.get(UserRegistry.java:594)

	at org.wso2.carbon.apimgt.impl.utils.APIUtil.getCustomSequence_aroundBody264(APIUtil.java:4589)

	... 68 more

[2020-09-18 18:41:50,769] ERROR - APIGatewayManager Error occurred deploying sequences on Production and Sandbox

org.wso2.carbon.apimgt.api.APIManagementException: Error in updating the sequence at the Gateway

	at org.wso2.carbon.apimgt.impl.APIGatewayManager.updateCustomSequences_aroundBody34(APIGatewayManager.java:851)

	at org.wso2.carbon.apimgt.impl.APIGatewayManager.updateCustomSequences(APIGatewayManager.java:798)

	at org.wso2.carbon.apimgt.impl.APIGatewayManager.publishToGateway_aroundBody2(APIGatewayManager.java:156)

	at org.wso2.carbon.apimgt.impl.APIGatewayManager.publishToGateway(APIGatewayManager.java:90)

	at org.wso2.carbon.apimgt.impl.APIProviderImpl.publishToGateway_aroundBody82(APIProviderImpl.java:1749)

	at org.wso2.carbon.apimgt.impl.APIProviderImpl.publishToGateway(APIProviderImpl.java:1733)

	at org.wso2.carbon.apimgt.impl.APIProviderImpl.updateAPI_aroundBody58(APIProviderImpl.java:1110)

	at org.wso2.carbon.apimgt.impl.APIProviderImpl.updateAPI(APIProviderImpl.java:978)

	at org.wso2.carbon.apimgt.impl.UserAwareAPIProvider.updateAPI_aroundBody6(UserAwareAPIProvider.java:88)

	at org.wso2.carbon.apimgt.impl.UserAwareAPIProvider.updateAPI(UserAwareAPIProvider.java:83)

	at org.wso2.carbon.apimgt.importexport.utils.APIImportUtil.updateAPI(APIImportUtil.java:672)

	at org.wso2.carbon.apimgt.importexport.APIService.updateAPI(APIService.java:352)

	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)

	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)

	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)

	at java.lang.reflect.Method.invoke(Method.java:498)

	at org.apache.cxf.service.invoker.AbstractInvoker.performInvocation(AbstractInvoker.java:179)

	at org.apache.cxf.service.invoker.AbstractInvoker.invoke(AbstractInvoker.java:96)

	at org.apache.cxf.jaxrs.JAXRSInvoker.invoke(JAXRSInvoker.java:193)

	at org.apache.cxf.jaxrs.JAXRSInvoker.invoke(JAXRSInvoker.java:103)

	at org.apache.cxf.interceptor.ServiceInvokerInterceptor$1.run(ServiceInvokerInterceptor.java:59)

	at org.apache.cxf.interceptor.ServiceInvokerInterceptor.handleMessage(ServiceInvokerInterceptor.java:96)

	at org.apache.cxf.phase.PhaseInterceptorChain.doIntercept(PhaseInterceptorChain.java:308)

	at org.apache.cxf.transport.ChainInitiationObserver.onMessage(ChainInitiationObserver.java:121)

	at org.apache.cxf.transport.http.AbstractHTTPDestination.invoke(AbstractHTTPDestination.java:267)

	at org.apache.cxf.transport.servlet.ServletController.invokeDestination(ServletController.java:234)

	at org.apache.cxf.transport.servlet.ServletController.invoke(ServletController.java:208)

	at org.apache.cxf.transport.servlet.ServletController.invoke(ServletController.java:160)

	at org.apache.cxf.transport.servlet.CXFNonSpringServlet.invoke(CXFNonSpringServlet.java:216)

	at org.apache.cxf.transport.servlet.AbstractHTTPServlet.handleRequest(AbstractHTTPServlet.java:301)

	at org.apache.cxf.transport.servlet.AbstractHTTPServlet.doPut(AbstractHTTPServlet.java:237)

	at javax.servlet.http.HttpServlet.service(HttpServlet.java:653)

	at org.apache.cxf.transport.servlet.AbstractHTTPServlet.service(AbstractHTTPServlet.java:276)

	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)

	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)

	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)

	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)

	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)

	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:219)

	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:110)

	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:603)

	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:169)

	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:104)

	at org.wso2.carbon.identity.context.rewrite.valve.TenantContextRewriteValve.invoke(TenantContextRewriteValve.java:80)

	at org.wso2.carbon.identity.authz.valve.AuthorizationValve.invoke(AuthorizationValve.java:100)

	at org.wso2.carbon.identity.auth.valve.AuthenticationValve.invoke(AuthenticationValve.java:65)

	at org.wso2.carbon.tomcat.ext.valves.CompositeValve.continueInvocation(CompositeValve.java:99)

	at org.wso2.carbon.tomcat.ext.valves.CarbonTomcatValve$1.invoke(CarbonTomcatValve.java:47)

	at org.wso2.carbon.webapp.mgt.TenantLazyLoaderValve.invoke(TenantLazyLoaderValve.java:57)

	at org.wso2.carbon.event.receiver.core.internal.tenantmgt.TenantLazyLoaderValve.invoke(TenantLazyLoaderValve.java:48)

	at org.wso2.carbon.tomcat.ext.valves.TomcatValveContainer.invokeValves(TomcatValveContainer.java:47)

	at org.wso2.carbon.tomcat.ext.valves.CompositeValve.invoke(CompositeValve.java:62)

	at org.wso2.carbon.tomcat.ext.valves.CarbonStuckThreadDetectionValve.invoke(CarbonStuckThreadDetectionValve.java:159)

	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:1025)

	at org.wso2.carbon.tomcat.ext.valves.CarbonContextCreatorValve.invoke(CarbonContextCreatorValve.java:57)

	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)

	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:445)

	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1137)

	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:637)

	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1775)

	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.run(NioEndpoint.java:1734)

	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)

	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)

	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)

	at java.lang.Thread.run(Thread.java:748)

Caused by: org.wso2.carbon.apimgt.api.APIManagementException: Issue is in accessing the Registry

	at org.wso2.carbon.apimgt.impl.utils.APIUtil.getCustomSequence_aroundBody264(APIUtil.java:4606)

	at org.wso2.carbon.apimgt.impl.utils.APIUtil.getCustomSequence(APIUtil.java:4547)

	at org.wso2.carbon.apimgt.impl.APIGatewayManager.deployInSequence_aroundBody28(APIGatewayManager.java:719)

	at org.wso2.carbon.apimgt.impl.APIGatewayManager.deployInSequence(APIGatewayManager.java:713)

	at org.wso2.carbon.apimgt.impl.APIGatewayManager.updateCustomSequences_aroundBody34(APIGatewayManager.java:829)

	... 64 more

Caused by: org.wso2.carbon.registry.core.exceptions.ResourceNotFoundException: Resource does not exist at path /_system/governance/apimgt/applicationdata/provider/admin/MyT/1.0.0/in/

	at org.wso2.carbon.registry.core.jdbc.EmbeddedRegistry.get(EmbeddedRegistry.java:532)

	at org.wso2.carbon.registry.core.caching.CacheBackedRegistry.getGhostResourceFromCache(CacheBackedRegistry.java:225)

	at org.wso2.carbon.registry.core.caching.CacheBackedRegistry.get(CacheBackedRegistry.java:198)

	at org.wso2.carbon.registry.core.session.UserRegistry.getInternal(UserRegistry.java:617)

	at org.wso2.carbon.registry.core.session.UserRegistry.access$400(UserRegistry.java:61)

	at org.wso2.carbon.registry.core.session.UserRegistry$5.run(UserRegistry.java:597)

	at org.wso2.carbon.registry.core.session.UserRegistry$5.run(UserRegistry.java:594)

	at java.security.AccessController.doPrivileged(Native Method)

	at org.wso2.carbon.registry.core.session.UserRegistry.get(UserRegistry.java:594)

	at org.wso2.carbon.apimgt.impl.utils.APIUtil.getCustomSequence_aroundBody264(APIUtil.java:4589)

	... 68 more

[2020-09-18 18:41:50,819]  INFO - CarbonAuthenticationUtil 'admin@carbon.super [-1234]' logged in at [2020-09-18 18:41:50,819+0530]

[2020-09-18 18:41:50,910]  INFO - CarbonAuthenticationUtil 'admin@carbon.super [-1234]' logged in at [2020-09-18 18:41:50,910+0530]

[2020-09-18 18:41:50,917] ERROR - APIService Error in updating API 

org.wso2.carbon.apimgt.importexport.APIImportException: Error in updating API 

	at org.wso2.carbon.apimgt.importexport.utils.APIImportUtil.updateAPI(APIImportUtil.java:706)

	at org.wso2.carbon.apimgt.importexport.APIService.updateAPI(APIService.java:352)

	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)

	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)

	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)

	at java.lang.reflect.Method.invoke(Method.java:498)

	at org.apache.cxf.service.invoker.AbstractInvoker.performInvocation(AbstractInvoker.java:179)

	at org.apache.cxf.service.invoker.AbstractInvoker.invoke(AbstractInvoker.java:96)

	at org.apache.cxf.jaxrs.JAXRSInvoker.invoke(JAXRSInvoker.java:193)

	at org.apache.cxf.jaxrs.JAXRSInvoker.invoke(JAXRSInvoker.java:103)

	at org.apache.cxf.interceptor.ServiceInvokerInterceptor$1.run(ServiceInvokerInterceptor.java:59)

	at org.apache.cxf.interceptor.ServiceInvokerInterceptor.handleMessage(ServiceInvokerInterceptor.java:96)

	at org.apache.cxf.phase.PhaseInterceptorChain.doIntercept(PhaseInterceptorChain.java:308)

	at org.apache.cxf.transport.ChainInitiationObserver.onMessage(ChainInitiationObserver.java:121)

	at org.apache.cxf.transport.http.AbstractHTTPDestination.invoke(AbstractHTTPDestination.java:267)

	at org.apache.cxf.transport.servlet.ServletController.invokeDestination(ServletController.java:234)

	at org.apache.cxf.transport.servlet.ServletController.invoke(ServletController.java:208)

	at org.apache.cxf.transport.servlet.ServletController.invoke(ServletController.java:160)

	at org.apache.cxf.transport.servlet.CXFNonSpringServlet.invoke(CXFNonSpringServlet.java:216)

	at org.apache.cxf.transport.servlet.AbstractHTTPServlet.handleRequest(AbstractHTTPServlet.java:301)

	at org.apache.cxf.transport.servlet.AbstractHTTPServlet.doPut(AbstractHTTPServlet.java:237)

	at javax.servlet.http.HttpServlet.service(HttpServlet.java:653)

	at org.apache.cxf.transport.servlet.AbstractHTTPServlet.service(AbstractHTTPServlet.java:276)

	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)

	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)

	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)

	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)

	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)

	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:219)

	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:110)

	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:603)

	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:169)

	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:104)

	at org.wso2.carbon.identity.context.rewrite.valve.TenantContextRewriteValve.invoke(TenantContextRewriteValve.java:80)

	at org.wso2.carbon.identity.authz.valve.AuthorizationValve.invoke(AuthorizationValve.java:100)

	at org.wso2.carbon.identity.auth.valve.AuthenticationValve.invoke(AuthenticationValve.java:65)

	at org.wso2.carbon.tomcat.ext.valves.CompositeValve.continueInvocation(CompositeValve.java:99)

	at org.wso2.carbon.tomcat.ext.valves.CarbonTomcatValve$1.invoke(CarbonTomcatValve.java:47)

	at org.wso2.carbon.webapp.mgt.TenantLazyLoaderValve.invoke(TenantLazyLoaderValve.java:57)

	at org.wso2.carbon.event.receiver.core.internal.tenantmgt.TenantLazyLoaderValve.invoke(TenantLazyLoaderValve.java:48)

	at org.wso2.carbon.tomcat.ext.valves.TomcatValveContainer.invokeValves(TomcatValveContainer.java:47)

	at org.wso2.carbon.tomcat.ext.valves.CompositeValve.invoke(CompositeValve.java:62)

	at org.wso2.carbon.tomcat.ext.valves.CarbonStuckThreadDetectionValve.invoke(CarbonStuckThreadDetectionValve.java:159)

	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:1025)

	at org.wso2.carbon.tomcat.ext.valves.CarbonContextCreatorValve.invoke(CarbonContextCreatorValve.java:57)

	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)

	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:445)

	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1137)
```

**Affected Product Version:**
APIM 2.6.0, API CLI 2.0.8

**Steps to reproduce:**

1. Create an API in env1.
2. Export it using the CLI tool.
3. Import it to the env2.
4. Update the API in the env1 by engaging a custom mediation in sequence.
5. Export the API again from env1.
6. Import the API again to the env2 using the --update flag.

**Comment by wasuradananjith** [comment]
The fix has been done and the documentation is updated [1] with the latest .war (**api-import-export-2.6.0-v17.war**). Hence closing the issue.

[1] https://docs.wso2.com/display/AM260/Migrating+the+APIs+and+Applications+to+a+Different+Environment

**Comment by wasuradananjith** [comment]
Fixed in APIMCLI 2.0.9 version (https://github.com/wso2/product-apim-tooling/releases/tag/v2.0.9)
